frontend/react/style-guide.md

```diff
@@ -932,33 +941,25 @@ src
   1. mapDispatchToProps (if using Redux)
   1. export default MyComponent
 
-  - How to define `propTypes`, `defaultProps`, `contextTypes`, etc...
+  - How to define `props`, `default values` etc...
 
-    ```jsx
+    ```tsx
     import React from 'react';
     import PropTypes from 'prop-types';
 
-    class Link extends Component {
-      static methodsAreOk() {
-        return true;
-      }
+    interface Props {
+      url: string;
+      id: number;
+      text: string;
+    };
 
-      render() {
-        const { url, id, text } = this.props;
-        return <a href={url} data-id={id}>{text}</a>;
+    function Link({ url, id, text }: Props) {
+      methodsAreOk() {
+        return true;
       }
+      return <a href={url} data-id={id}>{text}</a>;
     }
```

Comment:
what do you talk about in this section? i think that static methods is only in classes and if are a method inside the Link component, we need declare as arrow fuction to it work.
```suggestion
    function Link({ url, id, text }: Props) {
      const handleMethodsAreOk = () => {
        return true;
      }
      return <a href={url} data-id={id}>{text}</a>;
    }
```